src/patternfly/components/AppLauncher/app-launcher.scss

```diff
@@ -252,7 +252,7 @@
   height: var(--pf-c-app-launcher__menu-item-icon--Height);
   margin-right: var(--pf-c-app-launcher__menu-item-icon--MarginRight);
 
-  > * {
+  > img {
```

Comment:
Users can put whatever they want in there. Icons, svg's, img's, a `<picture>` element, whatever :)